core/src/com/mygdx/game/models/BrainstormingPhase.java

```diff
@@ -0,0 +1,62 @@
+package com.mygdx.game.models;
+import java.util.ArrayList;
+
+/**
+ * The BrainstormingPhase keeps track of the brains that have been fired and the wall.
+ *
+ * BRAIN_DAMAGE: How much damage 1 brain does to the wall
+ * Brains: The list of brains that has been sendt to the wall
+ * Wall: The wall
+ *
+ * This class implements the MVC pattern as a model.
+ *
+ * */
+
+public class BrainstormingPhase {
+
+    private final int BRAIN_DAMAGE;
+    private final Wall wall;
+    private ArrayList<Brain> brains = new ArrayList<>();
+
+    /**
+     * The constructor that takes the hitpointmax of the wall, and
```

Comment:
Very minor issue, but the variable in wall is called maxHitPoints, if we want to be consistent